tools/bazar/controllers/ApiController.php

```diff
@@ -173,6 +174,65 @@ public function createSemanticEntry($formId)
         ]);
     }
 
+    /**
+     * @Route("/api/bazar-list-data", methods={"GET"}, options={"acl":{"public"}})
+     */
+    public function getBazarListData()
+    {
+        try {
+            $bazatListService = $this->getService(BazarListService::class);
+            $forms = $bazatListService->getForms($_GET);
```

Comment:
```suggestion
            $forms = $bazatListService->getForms([]);
```

pour le moment, comme le mode `externalModeActivated` n'est pas possible pour bazarlsite dynamic, je ne vois pas l'intérêt de passer `$_GET`